### DISCUSSION TITLE
英文小写字母f出现问题

### DISCUSSION THREAD

**Discussion by tongysh** [atelier-anchor/smiley-sans#100] [General]
英文小写字母f，连续两个输入，即ff，出现问题，详解见下图：

<img width="536" alt="image" src="https://user-images.githubusercontent.com/54384675/211015444-47880983-0595-4300-92d0-50680a9a85aa.png">

**Comment by stone-zeng** [comment]
设计即是如此：

<img width="490" alt="Screenshot 2023-01-07 at 02 17 11" src="https://user-images.githubusercontent.com/18233088/211073721-7738d2e8-ddb2-4a32-af18-be9591d4c548.png">

这一特性通过 `calt` 控制，可以自行禁用（具体做法请参考软件或平台的相关文档）。

**Reply by tongysh** [reply]
ok
